conformance/tests/httproute-timeouts.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tests
+
+import (
+	"testing"
+
+	"k8s.io/apimachinery/pkg/types"
+
+	"sigs.k8s.io/gateway-api/conformance/utils/http"
+	"sigs.k8s.io/gateway-api/conformance/utils/kubernetes"
+	"sigs.k8s.io/gateway-api/conformance/utils/suite"
+)
+
+func init() {
+	ConformanceTests = append(ConformanceTests, HTTPRouteTimeoutRequest)
+	ConformanceTests = append(ConformanceTests, HTTPRouteTimeoutBackendRequest)
+}
+
+var HTTPRouteTimeoutRequest = suite.ConformanceTest{
```

Comment:
any way to instrument a retry so we can differentiate b/w `HTTPRouteTimeoutRequest` & `HTTPRouteTimeoutBackendRequest` ?

Reply:
I'm thinking that once we add retries to the API (future GEP-1731) we'll be able to add that kind of coverage. For now, just making sure both timeouts work.

Reply:
cool that works

Reply:
maybe you can add a TODO comment in there around this ?

Reply:
Good idea! Done.